common.go

```diff
@@ -14,7 +14,8 @@ import (
 // Binary name used for built package
 const binaryName = "watcher"
 
-var watcherFlags = []string{"run", "watch", "watch-vendor"}
+// watch-recursive works only for mac
+var watcherFlags = []string{"run", "watch", "watch-vendor","watch-recursive","watch-recursive-root"}
 
```

Comment:
for example: go run /Users/noamtzuberi/streamrail/go/src/github.com/noamtzuberi/go-watcher/cmd/watcher/main.go -watch-recursive true -watch-recursive-root github.com/streamrail